src/Http/Routing/src/Matching/ILEmitTrieFactory.cs

```diff
@@ -477,36 +477,38 @@ private class Labels
         public Label ReturnNotAscii { get; set; }
     }
 
-    private class Methods
+    private sealed class Methods
     {
         // Caching because the methods won't change, if we're being called once we're likely to
         // be called again.
         public static readonly Methods Instance = new Methods();
 
         private Methods()
         {
-            // Can't use GetMethod because the parameter is a generic method parameters.
-            Add = typeof(Unsafe)
-                .GetMethods(BindingFlags.Public | BindingFlags.Static)
-                .Where(m => m.Name == nameof(Unsafe.Add))
-                .Where(m => m.GetGenericArguments().Length == 1)
-                .Where(m => m.GetParameters().Length == 2)
-                .FirstOrDefault()
```

Comment:
Can the `using` for `Linq` now be removed❔

Reply:
Other parts of the file still use linq.